src/gpodder/util.py

```diff
@@ -218,6 +218,7 @@ def normalize_feed_url(url):
             # YouTube playlists. To get a list of playlists per-user, use:
             # https://gdata.youtube.com/feeds/api/users/<username>/playlists
             'ytpl:': 'http://gdata.youtube.com/feeds/api/playlists/%s',
+            'rtp:' : 'http://www.rtp.pt/play/podcast/%s'
```

Comment:
There's an RTP URL scheme, and why we don't yet support it (and may never support it), I'd rather not have this in here. Maybe "rtppt:" if you really want to have a shortcut? Or just leave it out.